File: examples/js/cache/get/getAValue.js
const { CacheService } = require("m3o/cache");

// Get an item from the cache by key
async function getAvalue() {
  let cacheService = new CacheService(process.env.MICRO_API_TOKEN);
  let rsp = await cacheService.get({
    key: "foo",
  });
  console.log(rsp);
}

getAvalue();

File: examples/go/cache/get/getAValue.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/cache"
)

// Get an item from the cache by key
func GetAvalue() {
	cacheService := cache.NewCacheService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := cacheService.Get(&cache.GetRequest{
		Key: "foo",
	})
	fmt.Println(rsp, err)
}
